bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -93,7 +93,9 @@ object BagsManager {
     }
   }
 
-  private class BagsThread(id: Int) extends Runnable {
+  private class BagsThread(id: Int) extends Thread {
+    setPriority(Thread.NORM_PRIORITY - 1)
```

Comment:
@swalker2m  had suggested as much, just to place the BAGS threads at a lower priority than the rest of the OT since they should be background tasks. I'm not sure if it will make a difference, and I haven't seen any performance issues, but I suspect there could potentially be performance issues on an older machine with a single CPU?